src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -1418,6 +1418,21 @@ namespace
         }
     }
 
+    const char* get_denoise_mode(const int denoise_mode)
+    {
+        switch (denoise_mode)
+        {
+        case 0:
```

Comment:
Half-indent `case` statements (2 more spaces). The biggest quirk of our formatting conventions :)